### PR TITLE
Remove duplicated links from the footer

### PR DESCRIPTION
**Pull request by garethwilliamsdesign**
### Trello card

[Trello 5087](https://trello.com/c/LGgaAufv)

### Context

The footer currently features a list of links that are duplicates of the top navigation. These links are rarely clicked and present issues through their duplication. Removing these links will clean up the footer and remove any issues associated with this duplication.

The only link from this list that should not be removed is ‘Search’. This is due to accessibility issues. As such, this link will be added to the list of links in the bottom half of the footer instead.

### Changes proposed in this pull request

Remove the duplicated links from the top half of the footer and add the 'Search' link to the list of links in the bottom half of the footer.

### Guidance to review

Check that the links in the top half of the footer have been removed and that 'Search' is now located alongside the links in the bottom half of the footer.

### CODE REVIEW

File: app/components/footer_component.html.erb [hunk 1]
```diff
@@ -38,19 +38,14 @@
...
-          <a href="<%= resource.path %>"><%= resource.title %></a>
-        <% end %>
-        <%= link_to "Find an event near you", events_path %>
-        <%= link_to "Sign up for updates", "/mailinglist/signup/name" %>
-        <%= link_to("Search", search_path) %>
       </div>
     </div>
     <div class="site-footer-bottom">
       <div class="site-footer-bottom__logo-container">
         <%= image_pack_tag 'static/dfelogo-white.svg', alt: "Department for Education", size: "124x73" %>
```
Comment (gemmadallmandfe) [inline app/components/footer_component.html.erb:45]:
```suggestion
        <%= image_pack_tag 'static/dfelogo-white.svg', alt: "Department for Education logo", size: "124x73" %>
```

I think we should change this alt text to be slightly more descriptive @garethwilliamsdesign (I know this is what it is in prod currently!)

Comment (gemmadallmandfe) [inline app/components/footer_component.html.erb:45]: When using a screen reader on VoiceOver, the focus order feels wrong - the DfE logo alt text is read out between the YouTube link and the Search link

(It is not tabbed to during focus order, just read out by screen readers)

I'm not sure whether you can do anything about this?

Comment (garethwilliamsdesign) [reply app/components/footer_component.html.erb]: I don't really understand why that would be the case as the code is clearly in a logical order. I'll take a look into this today.

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to go out on **Monday** (subject to a tech review)

**Pull request merged**